server/server.go

```diff
@@ -374,6 +374,11 @@ func (s *Server) StartExitMonitor() {
 	}
 	defer watcher.Close()
 
+	containerStoppedJobs := make(chan string, 100)
```

Comment:
Is 100 the right size for the buffer?  Should it be closer to the size of a containerid?

Reply:
that's the size of the buffered channel, not the string if that's what you're asking

Reply:
Thanks @runcom.  My question is should the channel size be roughly the same size as the anticipated string size?  100 just seems large to me,   But I don't know a ton about golang channels, so this is definitely an educational question for me.

Reply:
so that is the size of the messages which can be sent to the buffered channel. This line means "create a string channel which can contain 100 messages of string type" - it doesn't mean anything for the string length

Reply:
@runcom Ah, I see.  Thanks!  So we could have up to 100 strings representing the containerid in the channel at one time and then we'd start blocking once we had 100 there until the receiver processed some of them.  In that case we probably want to put 100 into crio.conf too.